dbt/seeds/location/schema.yml

```diff
@@ -0,0 +1,12 @@
+seeds:
+  - name: location.neighborhood_group
+    description: '{{ doc("seed_neighborhood_group") }}'
+    columns:
+      - name: nbhd
+        description: '{{ doc("shared_column_nbhd_code") }}'
+      - name: group_name
+        description: Name of the group that this neighborhood is in.
+      - name: updated_at
+        description: Timestamp of the last record update.
+      - name: version
+        description: Incrementing version of the record.
```

Comment:
The goal of this data model is to be flexible enough to allow us to adapt to speculative new features, without committing ourselves up front to features that turn out to be unnecessary. Some of the features that we have in mind are:

* We might need to change groupings based on feedback from Valuations or to reflect changes in the underlying markets. In that case, the `version` and `updated_at` columns will help preserve the history of the table so that we can always determine the exact set of groupings that were used to generate a set of flags without having to match timestamps on git commits.
* At some point we may need different groupings for res vs. condos. In this case, I expect we could add a `type` column to distinguish the two groups.
* We might decide to set a group to be equivalent to a township, as was the case in the original sales val runs. In that case, we could add a record for each neighborhood in a township using the same `group_name`.